src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -25,15 +28,29 @@ export default App.extend({
   },
   onStart({ patient }) {
     this.patient = patient;
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    this.canManagePatients = currentUser.can('patients:manage');
 
+    this.showSidebar();
+  },
+  showSidebar() {
     const widgets = Radio.request('bootstrap', 'sidebarWidgets');
 
     this.showView(new SidebarView({
       model: this.patient,
       collection: widgets,
+      canManagePatients: this.canManagePatients,
     }));
   },
   showPatientModal() {
     Radio.request('nav', 'patient', this.patient);
   },
+  toggleActiveStatus() {
+    this.patient.toggleActiveStatus();
+    this.showSidebar();
```

Comment:
When the status of the patient is changed, the text option in the menu and the status widget will need to re-render to show the updated status value.

Not sure how to re-render those individually... so I'm just re-rendering the entire sidebar to get those values to update. Seems okay to me, but open to a better way to do it.

Reply:
So on the status widget we can add an `initialize` where we get the `workspacePatient` and then just listenTo the `change:status` on that and re-render.

Reply:
That worked nicely 👍